src/helpers/historyExport.js

```diff
@@ -662,6 +663,57 @@ const getStartOfCourse = slotsGroupedByDate => (slotsGroupedByDate.length
 const isSlotInInterval = (slot, startDate, endDate) => CompaniDate(slot.startDate).isAfter(startDate) &&
   CompaniDate(slot.endDate).isBefore(endDate);
 
+const getAttendancesInfos = (course, attendanceSheetList) => {
+  const attendanceSheetsCount = attendanceSheetList
+    .filter(attendanceSheet => UtilsHelper.areObjectIdsEquals(attendanceSheet.course, course._id))
+    .length;
```

Comment:
ici tu passes toutes les feuilles d'émargement ce qui est un peu dommage, tu pourrais ne passer que celle de la formation...

De plus, `attendanceSheetsCount` n'est utilisée que dans le return, donc tu pourrais faire ce calcul dans la boucle for de la methode parente

Reply:
done